mne_sandbox/preprocessing/dss.py

```diff
@@ -0,0 +1,145 @@
+# -*- coding: utf-8 -*-
+
+"""Denoising source separation"""
```

Comment:
ahh, you should rename this module `_dss.py` because the import is getting screwed up by having a module `dss` and a `from .dss import dss` in `dss/__init__.py`